js/animation/fx.d.ts

```diff
@@ -1,10 +1,13 @@
-import {
-    DxElement
-} from '../core/element';
+import { DxElement } from '../core/element';
+import { DxPromise } from '../core/utils/deferred';
+import { positionConfig } from './position';
 
-import {
-    DxPromise
-} from '../core/utils/deferred';
+export type animationState = string | number | 
```

Comment:
@IlyaKhD @nightskylark I got rid of optional parameters so that we cannot pass an empty object now: even if it is possible, it seems to be useless since `from` and `to` fields are optional.